starlette/responses.py

```diff
@@ -37,11 +37,12 @@ def __init__(
         self.body = self.render(content)
         self.init_headers(headers)
 
-    def render(self, content: typing.Union[str, bytes, None]) -> bytes:
+    def render(self, content: typing.Any) -> bytes:
         if content is None:
             return b""
         if isinstance(content, bytes):
             return content
+        assert isinstance(content, str)
         return content.encode(self.charset)
```

Comment:
Maybe we can change it to this?

```py
def render(self, content: typing.Any) -> bytes:
        if isinstance(content, bytes):
            return content
        if isinstance(content, str):
            return content.encode(self.charset)
        return b""
```

It shouldn't have any breaking behaviour as we had limited content to be `str | bytes | None`.

Reply:
No. This can potentially hide issues for our users.

Reply:
yeah, maybe.